test/operator-convert.spec.ts

```diff
@@ -308,7 +308,7 @@ describe('convert operator unit tests', () => {
   });
 
   it('strings can be enumerated', () => {
-    const range = [...Array(101).keys()];
+    const range = Array.from({ length: 101 }).map((_, i) => i);
```

Comment:
Looks like `Array.from` is also not supported in IE11? (this is in test so probably fine but maybe it should be tested in IE1..?)

Reply:
I think I'll just make this test a for loop instead of building an array and `forEach`ing it

Reply:
Just used a for loop

![success](https://emoji.slack-edge.com/T02FE3LFK/success/fece95332fd73b56.png)